pkg/api/render.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/rebuy-de/kubernetes-deployment/pkg/templates"
 	log "github.com/sirupsen/logrus"
 	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/client-go/pkg/api"
+	"k8s.io/client-go/kubernetes/scheme"
```

Comment:
Such stable APU. Much wow.

Reply:
Can you check if this is still required: https://github.com/rebuy-de/kubernetes-deployment/blob/8c23e8d300e9706a2dbe7560411d4a76c9700f04/pkg/api/init.go#L8-L17

Reply:
Appears to no longer be needed.